### PR TITLE
string&RegExp&DateTime

### PR DESCRIPTION
**Pull request by AvetSargsyan**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+//1.Given the string 'ahb acb aeb aeeb adcb axeb'. Write a regular expression that matches the strings ahb, acb, aeb by pattern: letter 'a', any character, letter 'b'
+let str ='ahb acb aeb aeeb adcb axeb'.match(/a[a-z]b/g);
+console.log(str);
+
+//2.Given the string '2 + 3 223 2223'. Write a regexp that finds line 2 + 3 without capturing the rest
+let numStr = '2 + 3 223 2223';
+const reg =  /2...3/;
```
Comment (artiombond) [inline script.js]: it seems that "2  3", "2 - 3" and etc are valid?

File: script.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+console.log(numStr.match(reg));
+
+//3.Get the day, month and year of the current date and output it to the console separately
+const date = new Date();
+const now = [date.getDate(),date.getMonth(),date.getFullYear()]
+console.log(now);
```
Comment (artiombond) [inline script.js]: check please getMonth() function, for January it gives 0

File: script.js (hunk 1 above)
Comment (AvetSargsyan) [reply script.js]: I have changed it, is now everything right?

File: script.js (hunk 2 above)
Comment (AvetSargsyan) [reply script.js]: I didn’t take into account the fact that the count starts with 0 instead of 1.  Is now everything correct?

**Review by artiombond — APPROVED** [review]

**Pull request merged**